code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -347,13 +358,21 @@
 
 	return FALSE
 
-/obj/item/pinpointer/crew/attack_self(mob/living/user)
-	if(active)
-		active = FALSE
-		icon_state = icon_off
-		user.visible_message("<span class='notice'>[user] deactivates [user.p_their()] pinpointer.</span>", "<span class='notice'>You deactivate your pinpointer.</span>")
+/obj/item/pinpointer/crew/process()
+	if(mode == MODE_CREW && target_set)
+		point_at(target)
+
+/obj/item/pinpointer/crew/point_at(atom/target)
+	if(mode == MODE_OFF)
```

Comment:
Can be removed since the code before this should have tested this

Reply:
removed